### PR TITLE
Homepage

### PR DESCRIPTION
**Pull request by chanchal16**
- Created home page
- added filters
- filter by a category

https://user-images.githubusercontent.com/56486295/166295037-18c3e415-17bb-4d9c-8958-7128e7cff453.mp4

### CODE REVIEW

**Review by sandeepkumardev — APPROVED** [review]

File: src/components/CategoryCard.jsx [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+            <div className='content-title'>
+            <p className='text-lg'>{category.categoryName}</p>
+            </div>
+            <div className='desc'>
+            <span className='text-sm'>{category.description.substring(0,80)}..</span>
+            </div>
+        </div>                   
+    </div> 
+  )
+}
```
Comment (Sharmaryan) [inline src/components/CategoryCard.jsx:25]: destructuring from category will make your code more readable

File: src/index.css [hunk 2]
```diff
@@ -12,6 +12,55 @@ code {
...
+.explore:hover{
+  background-color: var(--accent);
+}
+.arrow{
+  vertical-align: middle;
+}
+/* categories */
+/* .categories{
+  padding: 1rem;
+} */
```
Comment (Sharmaryan) [inline src/index.css:50]: remove commented code

File: src/pages/Home.jsx [hunk 3]
```diff
@@ -0,0 +1,44 @@
+import React from 'react'
+import hero from '../assets/hero.svg';
+import {MdOutlineDoubleArrow} from 'react-icons/md';
+import { categories } from '../backend/db/categories';
+import { CategoryCard } from '../components';
+import { Link } from 'react-router-dom';
```
Comment (Sharmaryan) [inline src/pages/Home.jsx:6]: use [absolute path](https://dev.to/emanuelone/how-to-use-absolute-path-in-reactjs-503i#:~:text=Absolute%20Path%20is%20when%20you,located%20in%20the%20project%20structure.)

**Review by Sharmaryan — APPROVED** [review]

File: src/pages/Home.jsx (hunk 3 above)
Comment (chanchal16) [reply src/pages/Home.jsx:6]: As it's not deeply nested so I think I can keep it as it is for now

**Pull request merged**